lib/login_gov/hostdata.rb

```diff
@@ -28,7 +28,7 @@ def self.env
     end
 
     def self.instance_role
-      @env ||= begin
+      @instance_role ||= begin
```

Comment:
apparently this was me

![copy pasta](https://pics.me.me/lndo-cut-copy-past-delete-select-all-copypasta-its-delicious-39600288.png)